components/clipboard-button/index.js

```diff
@@ -77,14 +78,15 @@ class ClipboardButton extends Component {
 	render() {
 		// Disable reason: Exclude from spread props passed to Button
 		// eslint-disable-next-line no-unused-vars
-		const { className, children, onCopy, onFinishCopy, text, ...buttonProps } = this.props;
+		const { className, children, onCopy, onFinishCopy, text, icon = null, ...buttonProps } = this.props;
```

Comment:
Minor: I believe the default value is useless

Reply:
Oh  just saw your message, maybe we can avoid passing the icon entirely and just do something like: `ComponentToUse = buttonProps.icon ? IconButton : Button`